public/js/components/AtomCreate/AtomCreateGenericInfo.js

```diff
@@ -53,9 +54,28 @@ export class AtomCreateGenericInfo extends React.Component {
           />
         </div>
         <div className="create__buttons">
-          <button className="btn" onClick={this.createAtom}>Create Atom</button>
+          <button className="btn" onClick={this.triggerAtomCreate}>Create Atom</button>
         </div>
       </div>
     );
   }
 }
+
+//REDUX CONNECTIONS
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as createAtomActions from '../../actions/AtomActions/createAtom.js';
+
+function mapStateToProps(state) {
+  return {
+    config: state.config
```

Comment:
Is this required atm? If there's no additional overhead from having this then might as well leave it but just wondering.